src/Entity/FeaturedProgram.php

```diff
@@ -61,12 +61,40 @@ class FeaturedProgram
    * @ORM\Column(type="boolean", options={"default": false})
    */
   protected $for_ios = false;
+  /**
+   * @ORM\Column(type="string", options={"default": 0})
+   */
+  protected $language_version = 0;
```

Comment:
again.. why?

Reply:
Well I am opened to any suggestion how to do the query with language version. Because FeaturedProgram and Program are two different entities and Featured program doesn't have language version. How to join these?

Reply:
A snipped from the FeaturedProgram entity
```
  /**
   * @ORM\ManyToOne(targetEntity="Program", fetch="EAGER")
   */
  private $program;
```

you can access the project and all its members directly over the featured project.

```
    /** @var FeaturedProgram $p */
    $p->getProgram()->getLanguageVersion();
```